src/Infrastructure/Resources/config/services.yaml

```diff
@@ -584,6 +590,65 @@ services:
         class: SprykerSdk\Sdk\Infrastructure\Validator\Manifest\TaskSetManifestConfiguration
         arguments:
             - '@validation_helper'
+    manifest.interaction.need_to_ask_question:
+        class: SprykerSdk\Sdk\Infrastructure\Manifest\Interaction\Question\NeedToAskQuestion
+        arguments:
+            - "@cli_interaction_processor"
+
+    manifest.interaction.new_collection_item_question:
+        class: SprykerSdk\Sdk\Infrastructure\Manifest\Interaction\Question\NewCollectionItemQuestion
+        arguments:
+            - "@cli_interaction_processor"
+
+    manifest.interaction.value_question:
+        class: SprykerSdk\Sdk\Infrastructure\Manifest\Interaction\Question\ValueQuestion
+        arguments:
+            - "@cli_interaction_processor"
+
+    manifest.interaction.processor:
+        class: SprykerSdk\Sdk\Infrastructure\Manifest\Interaction\ManifestInteractionProcessor
+        arguments:
+            - "@manifest.interaction.need_to_ask_question"
+            - "@manifest.interaction.value_question"
+            - "@manifest.interaction.new_collection_item_question"
+
+    manifest.task_php_template_format_reader:
```

Comment:
`manifest.reader.task_php_template`. The same pattern for the rest of the readers and writers